rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/470_significant_texts.yml

```diff
@@ -21,9 +21,9 @@ setup:
         index: test
         refresh: true
         body:
-          - { "index": {} }
+          - { "index": { } }
```

Comment:
I think it's more normal not to have the spaces. But I doesn't matter either way.